src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -1018,12 +1030,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
-        Private ReadOnly Property IConversionExpression_IsExplicit As Boolean Implements IConversionExpression.IsExplicit
+        Private ReadOnly Property IConversionExpression_ExplicitCastInSource As Boolean Implements IConversionExpression.ExplicitCastInSource
             Get
                 Return Me.ExplicitCastInCode
             End Get
         End Property
 
+        Private ReadOnly Property IConversionExpression_Conversion As IConversion Implements IConversionExpression.Conversion
+            Get
+                Return New Conversion(Me.ConversionKind, method:=Nothing)
```

Comment:
Perhaps "inadequate testing" is more accurate than  "zero tests", because there are some analyzer tests that exercise IConversionExpression a bit, but I am the one guilty of checking in an inadequately-tested prototype. We are in the process of formalizing a plan for getting adequate testing, and I am not sure how heavy a requirement should be placed on a change like this. My inclination is to give a green light to a checkin that makes things better rather than require that it make things perfect, but the compiler team sets its own policies.

I will help with the testing and bug fixing after we come to agreement on the API changes.